test/new-e2e/agent-subcommands/health_test.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package agentsubcommands
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/e2e"
+	"github.com/stretchr/testify/assert"
+)
+
+type agentSuite struct {
+	e2e.Suite[e2e.AgentEnv]
+}
+
+func TestAgentHealthSuite(t *testing.T) {
+	e2e.Run(t, &agentSuite{}, e2e.AgentStackDef(nil))
```

Comment:
As this line of code creates and destroys a VM, I think it would make sense to test simple sub commands in the same `e2e.Suite`. What do you think?

Reply:
While I do agree we can save resources by merging multiple subcommands in one Suite, I believe status will have more complex tests in the future and might be moved into its own file to bring more clarity